crates/blockifier/src/scale_codecs.rs

```diff
@@ -0,0 +1,87 @@
+use cairo_vm::vm::runners::cairo_runner::ExecutionResources;
```

Comment:
Actually, you can also do a PR to our fork of the cairo-vm and derive/imp Encode/Decode on the struct directly. It will be useful to other people in the ecosystem.

(event if I like this impl)

Reply:
Done: https://github.com/keep-starknet-strange/cairo-rs/pull/3